doc/upgrade-cluster.md

```diff
@@ -23,29 +23,39 @@ Your Pravega cluster should be in a healthy state. You can check your cluster he
 ```
 $ kubectl get PravegaCluster
 NAME      VERSION   DESIRED MEMBERS   READY MEMBERS   AGE
-example   0.4.0     7                 7               11m
+pravega   0.4.0     7                 7               11m
 ```
 
 ## Upgrade Path Matrix
 
```

Comment:
Now that this is moved to a config map, lets remove this version-map from the documentation. Instead we could point to configmap.yaml

Reply:
done